src/services/edusharing/logic/connector.js

```diff
@@ -180,7 +180,9 @@ class EduSharingConnector {
 		if (this.isLoggedin() === false) {
 			await this.login();
 		}
-
+		// ES seems to expect a lowercase String
+		searchQuery = searchQuery.toLowerCase();
+		
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5979436)